### PR TITLE
2023 Call for Papers blog posts

### PR DESCRIPTION
**Pull request by lisushka**
Splitting them into two this year to make the main announcement easier to read.

### CODE REVIEW

File: _posts/2023-03-08-CFP-Open.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+For more information on talk topics and structure, go [here](/news/2023/03/08/CFP-Details).
+
+### What we expect from speakers
+
+The [SeaGL Code of Conduct](/code_of_conduct) applies to staff, presenters, volunteers, attendees, and sponsors alike.  The content of your presentation, and your behaviour at the conference, must abide by the Code of Conduct.
+
+To present at SeaGL, you'll need to be able to do one of the following:
+
+* be in Seattle on **November 3rd or 4th** to present live
+* submit a pre-recorded video to us by **October 1**
```
Comment (altsalt) [inline _posts/2023-03-08-CFP-Open.md:36]: maybe start with "_or_ "

File: _posts/2023-03-08-CFP-Open.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+### Office Hours
+SeaGL pioneered the idea of CfP Office Hours, so stay tuned for details, and if you need help in the meantime, please email us at {{ site.custom.a.email.cfp-help }}, and we'd love to help you work through a good proposal.
```
Comment (altsalt) [inline _posts/2023-03-08-CFP-Open.md:39]: change "details, and if you" to "details. If you". Also consider adding something like "or join SeaGL's General Discussion Matrix room".

File: _posts/2023-03-08-CFP-Open.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+### Who we are
+
+The Program Committee is the group responsible for selecting and scheduling all of the great talks you enjoy at SeaGL. This year, the Program Committee consists of:
+
+* Dawn Cooper (chair)
+* Ruth Ikegah
+
```
Comment (altsalt) [inline _posts/2023-03-08-CFP-Open.md:47]: Add something about still accepting new volunteers? Or maybe change the paragraph above to say that "the committee is chaired by".

File: _posts/2023-03-08-CFP-Open.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+We operate using the [SeaGL Program Committee Code of Practice](/code_of_practice).
+
+You can contact us via email at {{ site.custom.a.email.program }}, [Mastodon](https://mastodon.social/@SeaGL), or [Twitter](https://twitter.com/seagl).
```
Comment (altsalt) [inline _posts/2023-03-08-CFP-Open.md:50]: Add Matrix here, also there has been at least one call to drop the use of Twitter. I'm a bit mixed of opinion and cede to what you think for the CFP at least.

File: _posts/2023-03-08-CFP-Details.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+type: post
+published: true
+categories: news
+tags: '2023'
+---
+
+If you're thinking of submitting to SeaGL 2023, you've come to the right place to learn more!  For details on how to submit, go [here](/news/2023/03/08/CFP-Open).
+
+### Talk Categories
+We encourage almost any topic related to open source that you have a personal engagement with. We have created a list of topic ideas you might choose to use — these might give you some ideas.
```
Comment (altsalt) [inline _posts/2023-03-08-CFP-Details.md:14]: I think you mean to say something like, "We are broadly asking that talks fall into one of these categories. We've added examples of past talks for each to give you an idea of what we might be looking for."

File: _posts/2023-03-08-CFP-Details.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+  * [Building cloud networks: Terraform or Ansible?](https://archive.org/details/SeaGL2021-Francois_Caen-Building_cloud_networks_Terraform_or_Ansible), by François Caen and Troy Perkins at SeaGL 2021
+  * [Building Free CI/CD with GitHub Actions](https://archive.org/details/SeaGL2020-Cameron_Bielstein-Building_Free_CI_CD_with_GitHub_Actions), by Cameron Bielstein at SeaGL 2020
+  * [Microservices: A Cautionary Tale](https://archive.org/details/microservices_202001), by Amulya Bandikatla at SeaGL 2019
+- **Everything Else**: Have a great talk that doesn't fit these categories?  Submit it here!
+  * [The internet is unusable: the disabled view](https://archive.org/details/SeaGL2022-Nicolas_Steenhout-The_internet_is_unusable_The_disabled_view), by Nicolas Steenhout at SeaGL 2022
+  * [Patently Obvious](https://archive.org/details/SeaGL2020-Neil_McGovern-Patently_Obvious), by Neil McGovern at SeaGL 2020
+
+### What we are and aren't looking for
+
+SeaGL is a very special conference.  We're a community-focused Free/Libre Open Source Software annual event in Seattle, and since 2020, all over the world virtually!
```
Comment (altsalt) [inline _posts/2023-03-08-CFP-Details.md:54]: "free / libre / open source software, hardware, and culture" let's make sure it isn't JUST about software ;)

Maybe italicize "virtually"

File: _posts/2023-03-08-CFP-Details.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+  * [The internet is unusable: the disabled view](https://archive.org/details/SeaGL2022-Nicolas_Steenhout-The_internet_is_unusable_The_disabled_view), by Nicolas Steenhout at SeaGL 2022
+  * [Patently Obvious](https://archive.org/details/SeaGL2020-Neil_McGovern-Patently_Obvious), by Neil McGovern at SeaGL 2020
+
+### What we are and aren't looking for
+
+SeaGL is a very special conference.  We're a community-focused Free/Libre Open Source Software annual event in Seattle, and since 2020, all over the world virtually!  
+
+We're an independent bunch but we still like to take care of each other.
+
+We are not looking for sponsored talks, however we'd love to welcome you over at our [Sponsorship Prospectus](/sponsors/2022) and give you an opportunity to reach our attendees in other ways.  As a small community event, our attendees tend to be university students, open source hobbyists and engineers, security professionals, technical writers, and more, skewing toward community rather than a corporate feel.
```
Comment (altsalt) [inline _posts/2023-03-08-CFP-Details.md:58]: comma after "however".

Clearly this link should go to 2023... ASAP.

**Review by altsalt — COMMENTED** [review]
A few editing details, good job!

File: _posts/2023-03-08-CFP-Details.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (AndrewKvalheim) [reply _posts/2023-03-08-CFP-Details.md:58]: I’ve make the top-level [`/sponsors/`](https://seagl.org/sponsors/) page more usable so one option would be to just link to that.

**Pull request merged**